src/main/java/tsunami/security/scanner/utilities/App.java

```diff
@@ -16,12 +16,180 @@
+                        .endMetadata()
+                        .withNewSpec()
+                        .addNewContainer()
+                        .withName("www")
+                        .withImage("nginx")
+                        .withNewResources()
+                        .withLimits(new HashMap<>())
+                        .endResources()
+                        .endContainer()
+                        .endSpec()
+                        .build();
+        System.out.println(Yaml.dump(pod));
+
+        Map<String,String> m = new HashMap<>();
+        Map<String,String> m2 = new HashMap<>();
+        V1Service svc =
+                new V1ServiceBuilder()
+                        .withNewMetadata()
+                        .withName("wordpress")
+                        .withLabels(m)
+                        .addToLabels("app", "wordpress")
+                        .endMetadata()
+                        .withNewSpec()
+                        .addNewPort()
+                        .withPort(80)
+                        .endPort()
+                        .withSelector(m2)
+                        .addToSelector("app","wordpress")
+                        .addToSelector("tier", "frontend")
+                        .withType("LoadBalancer")
+                        .endSpec()
+                        .build();
+        System.out.println(Yaml.dump(svc));
+        */
+
+
+        // file path to KubeConfig
+        String kubeConfigPath = System.getProperty("user.home") + "/.kube/config";
+
+        // loading the out-of-cluster config, a kubeconfig from file-system
+        ApiClient client =
+                ClientBuilder.kubeconfig(KubeConfig.loadKubeConfig(new FileReader(kubeConfigPath))).build();
+
+        // set the global default api-client to the in-cluster one from above
+        Configuration.setDefaultApiClient(client);
+
+
+        // TODO: load kustomization or secret file (unfixed problem in creating namespaced secret object)
+        /*
+        String secretPath = System.getProperty("user.dir") + "/application/wordpress/mysql-pass.yaml";
+        File secretFile = new File(secretPath);
+        V1Secret secret = (V1Secret) Yaml.load(secretFile);
+        System.out.println(secret);
+        */
+
+        /* Initialize apis */
+        CoreV1Api api = new CoreV1Api();
+        AppsV1Api api1 = new AppsV1Api();
+
+        /* Load mysql config file */
+        String mysqlConfigPath = System.getProperty("user.dir") + "/application/wordpress/mysql-deployment.yaml";
+        File mysqlFile = new File(mysqlConfigPath);
+        List<Object> objectList4Mysql = (List<Object>) Yaml.loadAll(mysqlFile);
+
+        /* Load wordpress config file */
+        String wordpressConfigPath = System.getProperty("user.dir") + "/application/wordpress/wordpress-deployment.yaml";
+        File wordpressConfig = new File(wordpressConfigPath);
+        List<Object> objectList4Wordpress = (List<Object>) Yaml.loadAll(wordpressConfig);
```

Comment:
There is a comment on line 61 that you will load the config while traversing the file tree. So maybe move this block of code up to make it cleaner.